app/controllers/projects_controller.rb

```diff
@@ -1,6 +1,13 @@
+require 'octokit'
```

Comment:
This shouldn't need to be here, it's in the Gemfile so should be automatically required by bundler on startup